mriqc/cli/parser.py

```diff
@@ -197,14 +197,25 @@ def _bids_filter(value):
         "-n-cpus",
         action="store",
         type=PositiveInt,
-        help="Maximum number of threads across all processes.",
+        help="""\
+Maximum number of simultaneously running parallel processes executed by *MRIQC*
+(e.g., several instances of ANTs' registration).
+However, when ``--nprocs`` is greater or equal to the ``--omp-nthreads`` option,
+it also sets the maximum number of threads that simultaneously running processes
+may aggregate (meaning, with ``--nprocs 16 --omp-nthreads 8`` a maximum of two
+8-CPU-threaded processes will be running at a given time).
+Under this mode of operation, ``--nprocs`` sets the maximum number of processors
+that can be assigned work within an *MRIQC* job, which includes all the processors
+used by currently running single- and multi-threaded processes.""",
```

Comment:
Does this look okay in --help? It seems like a lot of text including markup.

Reply:
The markup is for sphinx-argparse - that gets rendered in https://mriqc.readthedocs.io/en/latest/running.html#command-line-interface.

I'm not particularly annoyed by this level of markup within the shell, if we can agree on that, I guess this is a thumb-up?

Reply:
Fair enough. :+1: